modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -37,18 +37,18 @@ master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
 worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
 ----
 
-. Change to the `eus-4.12` channel by running the following command:
+. Change to the `eus-4.10` channel by running the following command:
```

Comment:
Hmm, per [slack discussion](https://redhat-internal.slack.com/archives/CJ1J9C3V4/p1678810330149369), seems like we want to document EUS upgrade from 4.10 to 4.12 in 4.11 doc. cc @evakhoni to take a look.

Reply:
Sorry for the confusion - will update in Slack :)